adala/utils/parse.py

```diff
@@ -174,112 +208,104 @@ class ImageMessageChunk(TypedDict):
 Message = Union[str, List[MessageChunk]]
 
 
-class MessageChunkType(Enum):
-    TEXT = "text"
-    IMAGE_URL = "image_url"
-
-
 def split_message_into_chunks(
```

Comment:
Conducted some analysis, I propose not to roll back to the initial version.
Reason:
1. Performance: as the number of vars grows, the previous method shows degraded performance. The reason is due to repeated calls for partial_str_format() for each variable field, introducing linear complexity overhead
    ```
    split_message_into_chunks_v1: 331.67 ms avg
    split_message_into_chunks: 6.28 ms avg
    ```
2. More stable version: the previous version fails when encountering a non-existent field key
3. Control flow: `build_chunks` has multiple returns whereas new version is only one - bad practices according to single exit point pattern
4. Conciseness - less lines

Reply:
also added comments to improve readability